src/mainwindow/mainwindow.cpp

```diff
@@ -12,6 +12,7 @@
 #include <QFile>
 
 #include "localconnectorgenerator.h"
+#include "settingsconfigurator.h"
 #include "paletteconfigurator.h"
 #include "projectviewerdock.h"
 #include "documentmanager.h"
```

Comment:
use QT_END_NAMESPACE and QT_BEGIN_NAMESPACE

Reply:
It is cpp file